pontoon/sync/formats/xliff.py

```diff
@@ -96,6 +97,12 @@ def __init__(self, path, locale, source_resource=None):
                 # Add the entity to the entities dictionary using its key as the dictionary key
                 self.entities[entity.key] = entity
 
+            # Store target-language, needed for serialization
+            file_tag = self.xliff_file.namespaced("file")
+            file_nodes = self.xliff_file.document.getroot().iterchildren(file_tag)
+            if file_nodes:
+                self.target_language = next(file_nodes).get("target-language")
```

Comment:
Is it possible for this to be incorrectly set to the source language (i.e. `en-US`), as was discussed earlier?

Reply:
No, unless of course the l10n file is not set up properly in first place.